pkg/controller/v1beta1/inferenceservice/reconcilers/raw/raw_kube_reconciler.go

```diff
@@ -104,5 +109,5 @@ func (r *RawKubeReconciler) Reconcile() (*appsv1.Deployment, error) {
 	if err != nil {
 		return nil, err
 	}
-	return deployment, nil
+	return deployment[0], nil
```

Comment:
is there a change that the worker deployment is returned as first element in the array?

Reply:
I don't think so, worker deployment is always optional so it will be second all the time.

Reply:
If the default needs to be returned, it is better to return it filtering by name than by array index.

Reply:
ok I updated this part to make it clearer.